### PR TITLE
Reduce the number of warnings according to the test policy

### PR DESCRIPTION
**Pull request by o-kopysov**
Signed-off-by: Oleg Kopysov <o.kopysov@samsung.com>

# Description

Reduce the number of warnings according to the test policy.
The current number of warnings is 0.

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Code cleanup/refactoring
- [ ] Documentation update
- [ ] This change requires a documentation update
- [ ] CI system update
- [ ] Test Coverage update

# How Has This Been Tested?
```
$ mvn clean install 

[INFO] --- maven-compiler-plugin:3.10.1:compile (default-compile) @ lpvs ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 15 source files to D:\Git\GitHub\LPVS\target\classes
[INFO]
[INFO] --- maven-resources-plugin:3.2.0:testResources (default-testResources) @ lpvs ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Using 'UTF-8' encoding to copy filtered properties files.
[INFO] Copying 0 resource
[INFO]
[INFO] --- maven-compiler-plugin:3.10.1:testCompile (default-testCompile) @ lpvs ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 10 source files to D:\Git\GitHub\LPVS\target\test-classes

```

**Test Configuration**:
* Java: v11
* LPVS Release: v1.0.0

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by tdrozdovsky — APPROVED** [review]
LGTM

**Comment by o-kopysov** [comment]
@MoonkiHong  @t25kim 
I've completed working on warning fixing. Please take a look at this PR.

File: pom.xml [hunk 1]
```diff
@@ -65,6 +65,18 @@
             <version>4.7.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.findbugs</groupId>
+            <artifactId>annotations</artifactId>
+            <version>3.0.1</version>
+        </dependency>
+
+        <!-- https://mavenlibs.com/maven/dependency/com.infradna.tool/bridge-method-injector -->
```
Comment (MoonkiHong) [inline pom.xml:74]: Is there any motivation to leave this line as a comment?

**Review by MoonkiHong — COMMENTED** [review]
Please take a look at my comment.

File: pom.xml (hunk 1 above)
Comment (o-kopysov) [reply pom.xml:74]: No, I  will remove it.

**Review by MoonkiHong — APPROVED** [review]
+1

**Review by t25kim — APPROVED** [review]
LGTM!

**Pull request merged**